EKF/optflow_fusion.cpp

```diff
@@ -397,8 +397,6 @@ void Ekf::fuseOptFlow()
 			// run innovation consistency check
 			optflow_test_ratio[1] = sq(_flow_innov[1]) / (sq(math::max(_params.flow_innov_gate, 1.0f)) * _flow_innov_var[1]);
 
-		} else {
-			return;
```

Comment:
Looks like the loop has the following form:
```
for (uint8_t obs_index = 0; obs_index <= 1; obs_index++) {
  if (obs_index == 0) {
   X
  } else if (obs_index == 1) {
   Y
  }
}
```
w/o dependence on `obs_index` and thus can be simplified to:
```
{
  X
}
{
  Y
}
```